### ISSUE TITLE
React Native Support

### ISSUE THREAD

**Issue by killfish** [Radivarig/react-speed-reader#1]
I'd love to see support for React Native. Any plans to make this portable to the native platforms?

**Comment by Radivarig** [comment]
Yes, as it recently became available for Linux I didn't got a chance to test it yet. The engine (which is exported in the npm pkg) is a simple component, so I guess that only GUI should be adapted for Native.

**Comment by killfish** [comment]
Cool. Feel free to tag this as an enhancement. If you need any help porting the UI over just let me know.
